var/spack/repos/builtin/packages/root/package.py

```diff
@@ -234,6 +234,9 @@ class Root(CMakePackage):
     # See https://sft.its.cern.ch/jira/browse/ROOT-7517
     conflicts('%intel')
 
+    # ROOT v6.06 was incompatible with the GCC 5+ ABI
+    conflicts('%gcc@5.0.0:', when=':6.06.99')
```

Comment:
`':6.06.99'` -> `@:6.07.99` to fix error and per https://github.com/spack/spack/pull/14250#issuecomment-567969482.

Reply:
Fixed.